src/System.Private.ServiceModel/tools/IISHostedWcfService/Web.config

```diff
@@ -0,0 +1,95 @@
+        <add factory="WcfService.WebSocketHttpsDuplexTextStreamedTestServiceHostFactory" service="WcfService.WSDuplexService" relativeAddress="~\WebSocketHttpsDuplexTextStreamed.svc" />
+        <add factory="WcfService.WebSocketHttpRequestReplyTextStreamedTestServiceHostFactory" service="WcfService.WSRequestReplyService" relativeAddress="~\WebSocketHttpRequestReplyTextStreamed.svc" />
+        <add factory="WcfService.WebSocketHttpDuplexTextStreamedTestServiceHostFactory" service="WcfService.WSDuplexService" relativeAddress="~\WebSocketHttpDuplexTextStreamed.svc" />
+        <add factory="WcfService.WebSocketHttpRequestReplyTextBufferedTestServiceHostFactory" service="WcfService.WSRequestReplyService" relativeAddress="~\WebSocketHttpRequestReplyTextBuffered.svc" />
+        <add factory="WcfService.WebSocketHttpRequestReplyBinaryBufferedTestServiceHostFactory" service="WcfService.WSRequestReplyService" relativeAddress="~\WebSocketHttpRequestReplyBinaryBuffered.svc" />
+        <add factory="WcfService.WebSocketHttpDuplexTextBufferedTestServiceHostFactory" service="WcfService.WSDuplexService" relativeAddress="~\WebSocketHttpDuplexTextBuffered.svc" />
+        <add factory="WcfService.WebSocketHttpDuplexBinaryBufferedTestServiceHostFactory" service="WcfService.WSDuplexService" relativeAddress="~\WebSocketHttpDuplexBinaryBuffered.svc" />
+        <add factory="WcfService.WebSocketHttpsRequestReplyBinaryBufferedTestServiceHostFactory" service="WcfService.WSRequestReplyService" relativeAddress="~\WebSocketHttpsRequestReplyBinaryBuffered.svc" />
+        <add factory="WcfService.WebSocketHttpsRequestReplyTextBufferedTestServiceHostFactory" service="WcfService.WSRequestReplyService" relativeAddress="~\WebSocketHttpsRequestReplyTextBuffered.svc" />
+        <add factory="WcfService.WebSocketHttpsDuplexBinaryBufferedTestServiceHostFactory" service="WcfService.WSDuplexService" relativeAddress="~\WebSocketHttpsDuplexBinaryBuffered.svc" />
+        <add factory="WcfService.WebSocketHttpsDuplexTextBufferedTestServiceHostFactory" service="WcfService.WSDuplexService" relativeAddress="~\WebSocketHttpsDuplexTextBuffered.svc" />
+      </serviceActivations>
+    </serviceHostingEnvironment>
+  </system.serviceModel>
+  <system.webServer>
+     <modules runAllManagedModulesForAllRequests="true">
+        <add name="UrlRoutingModule" type="System.Web.Routing.UrlRoutingModule,             System.Web.Routing, Version=4.0.0.0,             Culture=neutral,             PublicKeyToken=31BF3856AD364E35" />
+    </modules>
+   <handlers>
+    <add name="UrlRoutingHandler" preCondition="integratedMode" verb="*" path="UrlRouting.axd" type="System.Web.HttpForbiddenHandler,            System.Web, Version=4.0.0.0, Culture=neutral,            PublicKeyToken=b03f5f7f11d50a3a" />
+   </handlers>
+    <directoryBrowse enabled="true" />
+        <staticContent>
+            <clientCache cacheControlMode="NoControl" />
+        </staticContent>
+  </system.webServer>
```

Comment:
I will fix the indent. 
I disabled the cache from UI, not sure why it's shown up as "No
control" instead of "DisableCache", I will update it manually.